src/renderer/components/MainPage.tsx

```diff
@@ -103,7 +103,7 @@ export default class MainPage extends React.PureComponent<Props, State> {
         this.threeDotMenu = React.createRef();
 
         this.state = {
-            key: this.props.teams.findIndex((team) => team.order === 0),
+            key: this.props.teams.length ? this.props.teams.findIndex((team) => team.order === 0) : 0,
```

Comment:
Can we add a comment here to notify people in future that key must be at least 0 (a valid array index), just so people know why it's like this

Reply:
actually, there is one option where it could be non-0 with this one, unlikely to happen but we can easily prevent it.
```js
const found = this.props.teams.findIndex((team) => team.order === 0);
const key = found >= 0 ? found : 0;
```

regarding the comment 0/5, it might help someone to read it, so it is fine :)

Reply:
If that issue happens, I'd like the app to crash because that means there is something else wrong with the configuration or our validation. If we check for it, it will fail silently and we may not notice the erroneous behaviour.